### PR TITLE
Remove configure option argument requirement as of PHP 8.1.0

### PR DESCRIPTION
**Pull request by alexandre-daubois**
Fix https://github.com/php/doc-en/issues/2102

### CODE REVIEW

File: reference/password/setup.xml [hunk 1]
```diff
@@ -20,8 +20,10 @@
   <para>
    To enable Argon2 password hashing, however, PHP must be build with libargon2
    support using the <option role="configure">--with-password-argon2[=DIR]</option>
-   configure option.
-  </para> 
+   configure option. As of PHP 8.1.0, the
+   <option role="configure">--with-password-argon2</option> option doesn't take
+   any argument.
+  </para>
```
Comment (Girgias) [inline reference/password/setup.xml:26]: I would reword this to make it future-proof by having the current way be the "main" and leading paragraph and then have something like:

> Prior to PHP 8.1.0, it was possible to specifiy the argon2 directory with ``command``

Comment (alexandre-daubois) [reply reference/password/setup.xml]: I updated the PR with your suggestion :slightly_smiling_face:

**Comment by Girgias** [comment]
Thank you
